rescue/2025-09-16/cosmic-helix/helix-renderer.mjs

```
@@ -0,0 +1,497 @@
+// Layer 2: Tree-of-Life nodes and paths; gentle strokes avoid harsh edges.
+function drawTreeOfLife(ctx, w, h, pathColor, nodeColor, N) {
+  ctx.save();
+  ctx.strokeStyle = pathColor;
+  ctx.lineWidth = N.TWENTYTWO / N.ELEVEN;
+  ctx.lineCap = "round";
+
+  const nodes = getTreeNodes(w, h);
+  const paths = getTreePaths();
+  ctx.lineWidth = N.TWENTYTWO / N.ELEVEN; // 22 paths, softened by pillar 11.
+
+  const nodes = [
```

Biome (2.1.2)

[error] Shouldn't redeclare 'nodes'. Consider to delete it or rename it.

'nodes' is defined here:


(lint/suspicious/noRedeclare)